Userland/Libraries/LibWeb/Loader/ResourceLoader.cpp

```diff
@@ -502,6 +507,13 @@ RefPtr<ResourceLoaderConnectorRequest> ResourceLoader::start_network_request(Loa
     if (!headers.contains("Accept-Encoding"))
         headers.set("Accept-Encoding", "gzip, deflate, br");
 
+    if (m_enable_client_hints) {
+        // TODO: These values should mirror the spoofed user agent
+        headers.set("sec-ch-ua", ByteString::formatted("\"Not/A)Brand\";v=\"8\", \"Chromium\";v=\"126\", \"Brave\";v=\"126\", \"{}\";v=\"{}\"", m_browser_name.to_byte_string(), m_browser_version.to_byte_string()));
```

Comment:
Where does the brand list here come from? I couldn't find a reference to the 'default brands' list. It seems odd to include the 'Not A Brand" and "Brave" entries here. We don't have either of those as spoofable user agents.

Reply:
I just copied what brave was sending to those sites to make them work. There is more info on why they are using the "Not/A)Brand" thing [here](https://wicg.github.io/ua-client-hints/#grease).

I'll action that TODO I've left there and make these reflect what the spoofed value is set to, although this will require  making some changes to the list of selectable user agents as it's not a 1 to 1 mapping of the standard user-agent.

Happy to take whatever action you think is appropriate.

Reply:
At least by default, Ladybird should claim to be Ladybird.

Reply:
The last part of that string format is adding Ladybird as the browser, Unfortunately doesn't work on it's own as the server just rejects it.

I did some further digging and neither firefox or safari support the [client hints](https://developer.mozilla.org/en-US/docs/Web/API/User-Agent_Client_Hints_API#api.navigatoruadata) API.
It looks like only chromium does and akamai is enforcing this when your user-agent is chrome/chromium.

I'm making a change to only enable the hints when it's chromium and making sure the versions match the spoofed values.